.github/workflows/tag-base-images.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
         run: |
           ./docker/registry/cpp-client-base/build/crane/retag.sh
```

Comment:
Should we remove cpp-client-base? Here, but also delete the registry directory and make sure nothing references it?

Reply:
r-base is based on cpp-client-base, so I don't think we can remove it from the base-images repo.   The tagging here I am not sure what it does, maybe we should have a short chat on slack or VC.